packages/distribution/translations/de.yaml

```diff
@@ -4,7 +4,12 @@ caluma:
     start: "Starten"
     send: "Offene Anfragen versenden"
     send-confirm: "Wollen Sie wirklich alle offenen Anfragen versenden?"
+    complete-confirm:
+      "Es gibt noch offene Anfragen in der aktuellen Zirkulation.
+      Wenn Sie die Zirkulation abschliessen, werden alle verbleibenden offenen
+      Anfragen abgebrochen. Möchten Sie fortfahren?"
```

Comment:
Let's tell them how many there are - we know it.

Reply:
Added the count and pluralization where necessary